deploy/helm/sumologic/conf/logs/logs.conf

```diff
@@ -3,5 +3,10 @@
   port 24321
   bind 0.0.0.0
 </source>
+<source>
```

Comment:
I'm confused why only `logs.conf` gets this liveness probe. Should this be moved to `common.conf` ?